### ISSUE TITLE
Docker image missing GLIBC

### ISSUE THREAD

**Issue by JoelBender** [oxigraph/oxigraph#723]
**Describe the bug**
Docker image tagged [latest](https://hub.docker.com/layers/oxigraph/oxigraph/latest/images/sha256-ead95076c640392ae2ab2b2bbae60d0e4087e281bdb84292c1725f89362b135e?context=explore) (uploaded Jan 2) fails to run.

**To Reproduce**
1. Which version of Oxigraph are you using? On which platform?
Ubuntu 22.04
```bash
$ uname -a
Linux canopus 6.2.0-39-generic #40~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Thu Nov 16 10:53:04 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
```

2. A command line or a code snippet that triggers the bug.

```bash
$ docker run -it --rm -v oxigraph-data:/data -p 7878:7878 oxigraph/oxigraph:latest --location /data serve --bind 0.0.0.0:7878
/usr/local/bin/oxigraph: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by /usr/local/bin/oxigraph)
/usr/local/bin/oxigraph: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by /usr/local/bin/oxigraph)
/usr/local/bin/oxigraph: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /usr/local/bin/oxigraph)
/usr/local/bin/oxigraph: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/local/bin/oxigraph)
/usr/local/bin/oxigraph: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /usr/local/bin/oxigraph)
```

Note that **oxigraph/oxigraph:v0.4.0-alpha.1** has the same problem, **oxigraph/oxigraph:v0.3.22** is fine.

**Comment by Tpt** [comment]
Hi! Sorry for that and thank you so much for the bug report. I was not expecting this alpha release to be tagged "latest". I will fix that and the glib C bug.

**Comment by Tpt** [comment]
I have moved the `latest` tag to point to 0.3.22.

**Comment by Tpt** [comment]
The glib error is fixed in 0.4.0-alpha.2
